src/pages/learn.jsx

```diff
@@ -32,19 +35,32 @@ export const learn = () => {
         <li>Souther Resident Killer Whale Call Catalog</li>
         <li>Exhibits</li>
       </ul>
+      <ThemeProvider theme={theme}>
```

Comment:
```suggestion
```

No need to add new ThemeProviders, there's already one higher up in the render tree ([see _app.tsx](https://github.com/orcasound/orcahome/blob/main/src/pages/_app.tsx#L28))

Reply:
Done @paulcretu